packages/runtime/src/widgets/__tests__/Button.test.tsx

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable react/no-children-prop */
+import { render, screen, fireEvent, waitFor } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import { BrowserRouter } from "react-router-dom";
+import { Button } from "../Button";
+import { createCustomWidget } from "../../runtime/customWidget";
+import { Column } from "../Column";
+
+describe("Button", () => {
+  test("test button loading using setLoading", async () => {
```

Comment:
You need to add a test where `loading` is initialized with a binding but then changed via `setLoading`